projects/views.py

```diff
@@ -135,15 +134,43 @@ def view_project(request, project_id):
                             request, f'User "{add_username}" added to project!'
                         )
                         form = ProjectAddUserForm()
-                except Exception as e:
+                except User.DoesNotExist:
+                    # Try sending an invite
+                    email_address = form.cleaned_data["user_ref"]
+                    try:
+                        validate_email(email_address)
+                        if email_exists_on_project(project, email_address):
+                            messages.error(request, "That email is tied to a "
```

Comment:
I wonder in what cases this is possible? So it would have to be a user that cannot be found in Portal by that email, yet the Keycloak IdP has that user mail on the project. Technically possible, I wonder if we have any cases in practice. We should probably keep the check, just thinking aloud.